internal/xds/rbac/rbac_engine.go

```diff
@@ -131,9 +139,37 @@ func newEngine(config *v3rbacpb.RBAC) (*engine, error) {
 		}
 		policies[name] = matcher
 	}
+
+	auditOptions := config.GetAuditLoggingOptions()
+	auditLoggers := []*audit.Logger{}
+	if auditOptions != nil {
```

Comment:
Suggest switch to
```
if auditOptions == nil {
    return ...
}
...
```
to reduce indentation for the more important block. This is common practice in Go, partially because Go chooses tabs over spaces.

Reply:
The thing is the engine is built during the return, so we'd add a whole other block for the return that looks like this

```
	return &engine{
		policies:       policies,
		action:         a,
		auditCondition: auditOptions.GetAuditCondition(),
		policyName:     policyName,
	}, nil
```

Reply:
We could build it above so it's like this, but I think it's messy in it's own right
```
...
...
engine := &engine{
		policies:       policies,
		action:         a,
		auditCondition: auditOptions.GetAuditCondition(),
		policyName:     policyName,
	}

if (auditOptions == nil) {
  return engine, nil
}

// do auditOptions parsing
engine.auditLoggers = auditLoggers
return engine, nil

```

Reply:
Alternatively, I could pull the whole parsing into it's own method which is probably the cleanest, then it can just return an empty auditLogger slice if there's nothing there.

Reply:
Changed to its own func, PTAL